charts/open-notificaties/templates/deployment.yaml

```diff
@@ -133,5 +139,79 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-
-
+---
+{{- if .Values.flower.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "open-notificaties.flowerFullname" . }}
+  labels:
+  {{- include "open-notificaties.flowerLabels" . | nindent 4 }}
+spec:
+  {{- if not .Values.flower.autoscaling.enabled }}
```

Comment:
```suggestion
```

Reply:
Ah, I was testing with a separate values.yaml which still contained `flower.autoscaling`, hence I did not encounter this error. 

Fixed by a6fb6dfabc9488a8c777bf2b684d21f16eb78e4b